Gemfile

```diff
@@ -17,8 +17,14 @@ gem "minima", "~> 2.5"
 group :jekyll_plugins do
   gem "jekyll-feed", "~> 0.12"
   gem "jekyll-remote-theme","~> 0.4.3"
+  gem "jekyll-include-cache"
+  gem "jekyll-paginate"
+  gem "jekyll-sitemap"
+  gem "jekyll-gist"
```

Comment:
Need to find out what each of these do, and either use them, or remove them in the long run I think